src/Parser/Warrior/Arms/Modules/Features/SpellUsable.js

```diff
@@ -0,0 +1,40 @@
+import SPELLS from 'common/SPELLS';
+import CoreSpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+const TACTICIAN_ID = SPELLS.TACTICIAN.id;
+const MORTAL_STRIKE_ID = SPELLS.MORTAL_STRIKE.id;
+const COLOSSUS_SMASH_ID = SPELLS.COLOSSUS_SMASH.id;
```

Comment:
no need to set these here, you can just use `SPELLS.COLOSSUS_SMASH.id` directly :)